src/style.css

```
@@ -0,0 +1,111 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(/fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(/fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  font-weight: 300;
+  font-family: Avenir, sans-serif;
+  box-sizing: border-box;
+}
+
+.form {
+  position: relative;
+  margin-top: 20px;
+}
+
+.input {
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  outline: 0;
+  width: 100%;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  color: #3d4e61;
+  font-weight: 800;
+}
+
+.bigInput {
+  height: 70px;
+  padding: 22px 0 23px 62px;
+  font-size: 16px;
+  line-height: 21.86px;
```

Don't just copy-pate styles from Figma
```suggestion
  line-height: 22px;
```